ydb/docs/en/core/devops/ansible/initial-deployment.md

```diff
@@ -100,10 +100,7 @@ Feel free to change these settings if needed, but it is not necessary in straigh
       - static-node-3.ydb-cluster.com
   ```
 
-The value of the `ydb_database_groups` variable in the `vars` section has a fixed value tied to the redundancy type and does not depend on the size of the cluster:
-
-* For the redundancy type `block-4-2`, the value of `ydb_database_groups` is seven.
-* For the redundancy type `mirror-3-dc`, the value of `ydb_database_groups` is eight.
+The value of the `ydb_database_groups` variable in the `vars` set the number of [storage groups](https://ydb.tech/docs/en/concepts/glossary#storage-group) for database. 
```

Comment:
Some guidance is still needed on what to write to that variable. Even newbies who don't have time to figure out our terminology, like storage groups, need to be able to be able to finish this instruction.